packages/fonts/src/font-data.ts

```diff
@@ -41,32 +22,47 @@ export const parseSubfamily = (subfamily: string) => {
       break;
     }
   }
-  let weight: WeightValue = weights.normal;
-  let possibleWeight: WeightKey;
-  for (possibleWeight in weights) {
-    if (subfamilyLow.includes(possibleWeight)) {
-      weight = weights[possibleWeight];
+
+  let weight: FontWeight = "400";
+  for (weight in fontWeights) {
+    const { name } = fontWeights[weight];
+    const { alt } = fontWeights[weight];
+    if (subfamilyLow.includes(name) || subfamilyLow.includes(alt)) {
       break;
     }
   }
-  return { style, weight };
+  return { style, weight: Number(weight) };
+};
+
+export const normalizeFamily = (family: string, subfamily: string) => {
+  let simplifiedFamily = family;
+  subfamily
+    .trim()
+    .split(" ")
+    .forEach((word: string) => {
+      simplifiedFamily = simplifiedFamily.replace(
+        new RegExp(word.trim(), "i"),
```

Comment:
Special regexp characters must be escaped in `word`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll — might be helpful

Reply:
you are right, I am using regex just because of case sensitivity option, I can rewrite it and not use regex